health_vara.py

```diff
@@ -29,6 +29,10 @@ def view_attributes(cls):
     def default_gender():
         return 'f'
 
+    @staticmethod
+    def default_name_representation():
+        return 'gf'
```

Comment:
Is there any documentation we can link to to better explain this?

Reply:
I've added a comment about the allowed values.